### DISCUSSION TITLE
Is there a best way to change domain for vaultwarden running in Docker on Ubuntu?

### DISCUSSION THREAD

**Discussion by dervish54** [dani-garcia/vaultwarden#4536] [Q&A]
I have a need to change the domain for vaultwarden. Normally, I would run a new copy with a new domain. However, I wondered if there are proven ways in the community to replace the domain in the same copy without creating a new one.

**Comment by jobritz** [comment]
If you are referring to the DOMAIN enviromanet variable, have you tried this: https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page ? The Domain is the first setting there

**Reply by auspex** [reply]
Unfortunately not useful if you made the mistake of changing the domain _first_. I'm sure it can be done either by changing a config file or through the CLI, but I came here looking for that info.

**Reply by hiyan** [reply]
There should be a config.json in the data folder. Settings can be change there.
